pvlib/singlediode_methods.py

```diff
@@ -62,7 +64,9 @@ def estimate_voc(photocurrent, saturation_current, nNsVth):
 
 
 def bishop88(diode_voltage, photocurrent, saturation_current,
-             resistance_series, resistance_shunt, nNsVth, gradients=False):
+             resistance_series, resistance_shunt, nNsVth, d2mutau=0,
+             cells_in_series=None, voltage_builtin=VOLTAGE_BUILTIN,
```

Comment:
A nit - re-order the arguments to keep the non-recombination current terms first, followed by `d2mutau` and `voltage_builtin` - just move `cells_in_series` up one position.

Reply:
✔️